README.md

```diff
@@ -17,6 +17,16 @@ The OWASP Zed Attack Proxy (ZAP) is one of the world’s most popular free secur
 
 #### Please help us to make ZAP even better for you by answering the [ZAP User Questionnaire](https://docs.google.com/forms/d/1-k-vcj_sSxlil6XLxCFade-m-IQVeE2h9gduA-2ZPPA/viewform)!
 
+#### ZAP in Ten - Video series
+An ongoing series of up to 10 minute videos about ZAP, starting with the basics. 
+
+The first episode and the full series are available via:
+
+[![](https://play.vidyard.com/RyTy22GZV6UccW41UCghC8.jpg?email=1)](http://play.sonatype.com/watch/RyTy22GZV6UccW41UCghC8?)
```

Comment:
This doesn't work for some reason:
![image](https://user-images.githubusercontent.com/7570458/69659195-ae3c4b00-104b-11ea-9bcd-00110b2cd583.png)

Reply:
Could be a problem with https://alldaydevops.com/zap-in-ten ? - that was working for me but not right now :/

Reply:
Okay, I'll approve and merge. We can always re-tweak later if necessary.